### ISSUE TITLE
Building the project myself... 

### ISSUE THREAD

**Issue by mtallenca** [vaadin/gwt-polymer-elements#93]
When running npm install, the latest version of bower components are not necessarily installed.  The vaadin components are specifying a particular version.  i.e. iron-list latest version is 1.1.7 and vaadin-grid forces install of 1.0.3.  

If I remove the 3 vaadin components from the root packages.json file, I get the latest iron-list.  

Do the vaadin components need a specific version of the components ?

**Comment by manolo** [comment]
That is an issue in vaadin-combo-box which depends on it, I don't see any reason to use latest though, asking to the team.

**Comment by manolo** [comment]
Uploaded 1.2.3.1-SNAPSHOT with that fixed, could you @mtallenca test it in the meanwhile we fix it in vaadin-combo-box ?

**Comment by mtallenca** [comment]
Just did a pull and built 1.2.3.1-SNAPSHOT.  iron-list.html is still old version 1.0.3.  One reason to use the latest is iron-list 1.2 (latest) has method "updateSizeForItem" which doesn't exist in 1.0.3.  The method allows me to make row-specific html updates (specifically turn newlines into br tags) to iron-list and inform iron-list to resize a single item.

**Comment by manolo** [comment]
Something seems wrong, in your build can you do the following:

```
$ git pull origin master
$ rm -rf node_modules pom.xml src
$ npm install
```

then in src/main/resources..../iron-list/bower.json should be the latest

**Comment by mtallenca** [comment]
I'm still seeing 1.0.3 :(  I've attached the Terminal output as log.txt.gz

if I run

sed -i '' -e 's/,vaadin\/vaadin-combo-box#master//' package.json

Then after 'npm install' I see version 1.2.2 of iron-list

Thanks again for looking into this,

Mike

**Comment by manolo** [comment]
Fixed in 1.9.3.0
